kolibri/plugins/learn/assets/src/vue/content-render.vue

```diff
@@ -0,0 +1,47 @@
+<template>
```

Comment:
* I think if we want it available across everything (and given the need for coach report preview mode, we do want that), then yes. Can move it!
* `content-renderer` seems a far better name!